### ISSUE TITLE
CLB: Sectors shown as broken in the CoL project - Sectors report, but they appear as healthy in the Assembly.

### ISSUE THREAD

**Issue by yroskov** [CatalogueOfLife/checklistbank#1241]
@mdoering & @thomasstjerne, I bring up the same problem again: sectors shown as broken in the CoL project - Sectors report, but they appear as healthy in the Assembly.

![image](https://github.com/CatalogueOfLife/data/assets/28574252/917e59ec-f2c0-4e97-8d21-74e7284a432c)
![image](https://github.com/CatalogueOfLife/data/assets/28574252/3de635c0-1468-42c1-9ba7-2b89eac4d047)
![image](https://github.com/CatalogueOfLife/data/assets/28574252/1a3a7107-2ccc-4e5c-8cd2-9510549673f9)

![image](https://github.com/CatalogueOfLife/data/assets/28574252/79658fe3-acdc-4625-85fc-f326cf226643)
![image](https://github.com/CatalogueOfLife/data/assets/28574252/bcab8c73-4e84-4680-8131-9978e8791b3f)
![image](https://github.com/CatalogueOfLife/data/assets/28574252/b378f302-e165-46ea-9e28-952f05e57e2a)

_Originally posted by @yroskov in https://github.com/CatalogueOfLife/data/issues/535#issuecomment-1572664217_

**Comment by mdoering** [comment]
Yes, because they are broken on the _target_ side, not the subject.
The assembly tree does not use the target and therefore doesn't show it. It shows the current existing data instead.
This is expected behavior and I am not sure what else you would want to see here

**Comment by yroskov** [comment]
> I am not sure what else you would want to see here

Simple. If sector is broken, it should be shown as broken everywhere. Sector is always a **link** between chosen child in the GSD and the parent in CoL management classification. If this link is broken, it must be shown as broken.

**Comment by mdoering** [comment]
But the assembly tree does not work like that I am afraid. The tabular sector list shows it that way, but the assembly tree must link ALL NAMES that CAME from the last sector sync. It works like the tree browser on the COL portal with its info box. So the target link is nowhere shown.
